code-for-canada/project-thundercat
# [Organizational Structure] Populating organizational structure tree view with api

# Description

This PR is introducing new API calls to get and populate the tree view in _Organizational Structure_ image description.

**TODO(1):** Populate the popup box description of the image description with markdown content (see PR #426 )
**TODO(2):** Find a way to convert _treeView_ variable to a local state

## Type of change

Please delete options that are not relevant.

- New feature (non-breaking change which adds functionality)
- Code cleanliness or refactor

## Screenshot

**Tree View (EN):**

![image](https://user-images.githubusercontent.com/23021242/60662228-c9eb4e80-9e29-11e9-861f-fb1847349f71.png)

**Tree View (FR):**

![image](https://user-images.githubusercontent.com/23021242/60662244-d53e7a00-9e29-11e9-82c2-3ac1a1b069a0.png)

# Testing

Manual steps to reproduce this functionality:

1.  Start the eMIB Sample Test.
2.  Click on _Background_ tab.
3.  Click on _Organizational structure_ on side nav.
4.  Open the image description.
5.  Make sure that the tree view looks exactly the same as before in both languages.

# Checklist
